dev/Common/ThicknessFilterConverter.cpp

```diff
@@ -0,0 +1,61 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+#include <pch.h>
+#include <common.h>
+#include "ThicknessFilterConverter.h"
+
+winrt::Thickness ThicknessFilterConverter::Convert(winrt::CornerRadius const& radius, winrt::ThicknessFilterConverterKind const& filterKind)
```

Comment:
*May be* we can just add a property (like "ThicknessMode") to CornerRadiusFilterConverter and make it return Thickness instead of creating a new converter?